build-system/server.js

```diff
@@ -56,9 +56,9 @@ setInterval(function() {
   }
 }, 1000);
 
-const middleware = [];
+const middleware = [app];
 if (!quiet) {
-  middleware.push(morgan('dev'), app);
+  middleware.push(morgan('dev'));
```

Comment:
Did you make sure that it's only `morgan('dev')` that's responsible for the URL switching?